app/components/search-facet-typeahead/component.js

```diff
@@ -49,8 +66,14 @@ export default Ember.Component.extend({
     },
 
     actions: {
-        changeFilter(selected) {
-            this.sendAction('onChange', this.get('key'), this.buildQueryObject(selected));
+        changeFilter(selected, match=true) {
```

Comment:
I couldn't find anywhere this is called with `match=false`, do we need `match` at all? Or `buildQueryObjectCombine`?